django/utils/feedgenerator.py

```diff
@@ -223,7 +223,7 @@ def latest_post_date(self):
                     if latest_date is None or item_date > latest_date:
                         latest_date = item_date
 
-        return latest_date or datetime.datetime.now()
+        return latest_date or datetime.datetime.now(tz=timezone.utc)
```

Comment:
Django has already a helper to do the same thing (with addition of checking if settings.USE_TZ is True) at django/utils/timezone.py: django.utils.timezone.now.